bundlewrap/node.py

```diff
@@ -218,6 +224,7 @@ def handle_result(task_id, return_value, duration):
                     Item.STATUS_SKIPPED,
                     interactive=interactive,
                     details=Item.SKIP_REASON_DEP_FAILED,
+                    show_skipped_items=show_skipped_items,
```

Comment:
Aus meiner sicht ist das nicht der richtige Weg – wir *wissen* ja hier dass der status `Item.STATUS_SKIPPED` ist weil wir es fest reingeben und wir _wissen auch_, dass `handle_apply_result` dann eine NOOP ist, also müssen wir eigentlich garnicht erst da rein callen. Ich fände es schöner hier an der Call-Site zu prüfen.

Hier wäre es mit einem `if not show_skipped_items: handle_apply_result(…)` getan, ebenso in Zeile 242 und 278. In Zeile 288 wissen wir dass `show_skipped_items` immer egal ist, weil wir fest `Item.STATUS_FAILED` als Code reingeben. Einzig Zeile 259 hat eine variablen Statuscode und dort sollte mMn. dann auch dieser geprüft werden.

Reply:
This is a matter of taste.

1.  Cluttering the call sites with more `if`s makes it harder to read/follow, imo. 
2.  `handle_apply_result()` already has an `if` that skips printing in some circumstances. Adding another condition for `show_skipped_items` felt natural. Now we have two places where decisions are being made (call site and called function).

I pushed the changes as requested, not too happy with it, but I also don’t want to lose time arguing over this, so fine by me. ✅